packages/itwin/grouping-mapping-widget/package.json

```diff
@@ -62,26 +62,25 @@
     "simple-react-validator": "^1.6.1"
   },
   "devDependencies": {
-    "@itwin/appui-abstract": "^3.0.0",
-    "@itwin/appui-layout-react": "^3.0.0",
-    "@itwin/appui-react": "^3.0.0",
-    "@itwin/build-tools": "^3.0.0",
-    "@itwin/components-react": "^3.0.0",
-    "@itwin/core-bentley": "^3.0.0",
-    "@itwin/core-common": "^3.0.0",
-    "@itwin/core-frontend": "^3.0.0",
-    "@itwin/core-geometry": "^3.0.0",
-    "@itwin/core-i18n": "^3.0.0",
-    "@itwin/core-markup": "^3.0.0",
-    "@itwin/core-orbitgt": "^3.0.0",
-    "@itwin/core-quantity": "^3.0.0",
-    "@itwin/core-react": "^3.0.0",
-    "@itwin/eslint-plugin": "^3.0.0",
-    "@itwin/itwinui-css": "^0.50.0",
-    "@itwin/presentation-common": "^3.0.0",
-    "@itwin/presentation-components": "^3.0.0",
-    "@itwin/presentation-frontend": "^3.0.0",
-    "@itwin/webgl-compatibility": "^3.0.0",
+    "@itwin/appui-abstract": "^3.1.0",
```

Comment:
Did you need to bump all your deps to 3.1?

Reply:
Yes, appui-react 3.1.0 has the latest itwinui-react dependency. Also doing rush update -full is risky and breaks the other widgets in this repo. Unless there is a better way of doing this.

Reply:
I just noticed your other comment. Is that why the other widgets break when I do rush update -full? I can undo this if that is the case and wait for https://github.com/iTwin/itwinjs-core/pull/3446

Reply:
Yea thats why other pkgs broke, i'd revert your commit and wait for 3.1.1